.codeclimate.yml

```diff
@@ -0,0 +1,16 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  eslint:
+    enabled: true
+ratings:
+  paths:
+  - "**.js"
+  - "**.jsx"
+exclude_paths:
+- dist/
+- test/
```

Comment:
@bartriordan Unless the support tools are used by the production code (in which case they shouldn't be in a test directory), I'm not in favor of having something like CodeClimate rate our tests. I'd like tests to be well-written, but we're most concerned about the quality of code that goes out into production and CC is useful for prioritizing what modules we should work on repairing.